tests-ng/_test_dummy.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+
+def test_always_pass():
+	assert True
+
+@pytest.mark.feature("nonExistingFeature")
+def test_feature_never():
+	assert False
+
+@pytest.mark.feature("base")
+def test_feature_base():
+	assert True
+
+@pytest.mark.feature("(cloud or metal) and not _usi")
+def test_rootfs_ext4_options():
+	# perform some check here
+	pass
+
+@pytest.mark.root
+def test_root_user(shell):
+	assert shell("id -u", capture_output=True).stdout.strip() == "0"
+
+def test_regular_user(shell):
+	assert shell("id -u", capture_output=True).stdout.strip() != "0"
+
+def test_shell_print(shell):
```

Comment:
general comment/question: is there a good reason against using type hintes here? (and in all other places in the framework)? I think this would improve usability for developers a lot so it is easy to discover what the type of this thing is and what methods/properties it has.

the old framework seems to use type hints